index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   "ғ": "ğ",
   "д": "d",
   "е": "e",
-  "ё": "ıo",
+  "ё": "ö",
```

Comment:
> §78. ё әрпі о́ әрпімен таңбаланады

i think `ö` is not `о́`, because we have `"ө": "ö",`

Reply:
no, no. `ө` used to be `о́ ` in the first version of latin alphabet. The paragraph you are referring to is based on that. Later it became `ö`, So `ё` and `ө` are both `ö`. 

Logic is simple. Since `ё` is not native sound in kazakh language, it is used only in imported words. And imported words should be adapted to kazakh language. Same with `ц`